packages/shared/src/utils/config/platform-config.js

```diff
@@ -8,4 +8,11 @@ export const platforms = {
         route_to_path: routes.cashier_p2p,
```

Comment:
We should introduce this as a Context value in PlatformContext. It is especially designed to avoid prop-drilling. Would you please create a TODO comment for this?

Reply:
will do 🙏🏼